packages/libaccessom2/package.py

```diff
@@ -38,6 +38,14 @@ class Libaccessom2(CMakePackage):
     def url_for_version(self, version):
         return "https://github.com/ACCESS-NRI/libaccessom2/tarball/{0}".format(version)
 
-    def cmake_args(self):
```

Comment:
Just a quick check: did anything call this function that would miss it being deleted?

Reply:
My understanding is that is a function over-ride mechanism as described in https://spack.readthedocs.io/en/latest/build_systems/cmakepackage.html#adding-flags-to-cmake .

Reply:
Noted.